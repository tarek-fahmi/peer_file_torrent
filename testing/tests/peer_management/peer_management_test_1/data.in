Peer Connection Attempt
./testing/bin/btide ./testing/resources/configs/config.cfg
CONNECT 127.0.0.1:9000
CONNECT 127.0.0.1:9000
QUIT