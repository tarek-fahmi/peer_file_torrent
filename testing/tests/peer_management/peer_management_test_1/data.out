Connection established with peer
Already connected to peer
